### PR TITLE
Feat/FRO-83-add_people_to_channel_functionality

### PR DESCRIPTION
**Pull request by willhemz**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-FRO-83**

# Changes proposed

## What were you told to do?

Fix the add people to channel functionality

## What did you do?

- Message panel styling was fixed by setting the state value of sidebarWidth from 150 to 260.
- The response fetched from an API call to the server containing the users in the workspace are then set to the userList state value.
- on clicking the select tab, a list of users then displays.

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

https://user-images.githubusercontent.com/104460178/204160715-64cd5626-a5e9-4e54-b97b-17a61da48ca5.mp4

### CODE REVIEW

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const Sidebar = props => {
 
   const sidebarRef = useRef(null);
   const [isResizing, setIsResizing] = useState(false);
-  const [sidebarWidth, setSidebarWidth] = useState(150);
+  const [sidebarWidth, setSidebarWidth] = useState(240);
```
Comment (billmal071) [inline packages/main/src/components/protected/sidebar/components/Sidebar.jsx]: Did you pull, I don't think you need to touch the sidebar again. @JulianaSau

**Review by billmal071 — CHANGES_REQUESTED** [review]
Pull again, some changes here are stale.

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx (hunk 1 above)
Comment (JulianaSau) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx]: @willhemz please don't touch the sidebar🥲. We just fixed it

**Comment by JulianaSau** [comment]
I also know that this functionality was already implemented by another intern in stage 4. How is this PR adding to this functionality?

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx (hunk 1 above)
Comment (willhemz) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx]: Alright. I'll do that now. Thanks

Comment (willhemz) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx]: @billmal071 @JulianaSau my branch is updated with recent changes. Kindly re-review. thanks

**Comment by ickynavigator** [comment]
your PR has only one change👀

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx (hunk 1 above)
Comment (willhemz) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx]: @JulianaSau @billmal071 I've added the functionality, such that members already in the channel will not display in the search option.

**Pull request closed without merging**